src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -152,16 +148,16 @@ public MainMenu() {
 		}
 
 		if (singleUser) {
-			LOGGER.info("Logging: Single User mode.");
-			myUser = new User(ADMIN_STR, new UserGroup(ADMIN_STR, ""), ADMIN_STR, "");
+			logger.info("Logging: Single User mode.");
+			myUser = new User("admin", new UserGroup("admin", ""), "admin", "");
```

Comment:
correct form is with LOGGER and constants

Reply:
And why undo the "ADMIN_STR"?; that saves allocation of string literals and makes changing the value easier if ever needed.

Reply:
I restored the current informatici version